contracts/lbp/LBPWrapper.sol

```diff
@@ -18,11 +18,13 @@ import "../utils/interface/IVault.sol";
 import "../utils/interface/ILBP.sol";
 
 contract LBPWrapper {
+    uint256 public constant SWAP_FEE_PERCENTAGE = 1e12; // 0.0001% is minimum amount.
```

Comment:
comment-> ```// 0.0001% is minimum swap fee allowed by LBP```.

Reply:
also, let's add this to WrapperFactory, because someday, we might want to change it as mention by Luuk on latest workshop. LBPWrapperFactory will pass the swap fee to LBPWrapper.